### PR TITLE
Clean up Art/Models+Sound

### PR DESCRIPTION
**Pull request by cosmiccoincidence**
## Summary

Cleans up the Models and Sound folders under the Art folder.

- Renames a couple files using PascalCase.
- Fixes missing material links in model metadatas.
- Sorts Sound folder.

### CODE REVIEW

**Review by stilnat — DISMISSED** [review]
wait for #995 before merge.

**Comment by cosmiccoincidence** [comment]
Wait for #1021 to merge first!

**Review by TaylorNAlbarnaz — APPROVED** [review]
lgtm

**Pull request merged**
